examples/machine-learning/a3-highgpu-8g/ml-slurm-a3-1-image.yaml

```diff
@@ -92,7 +94,7 @@ deployment_groups:
           set -e -o pipefail
           ansible-galaxy role install googlecloudplatform.google_cloud_ops_agents
           ansible-pull \
-              -U https://github.com/GoogleCloudPlatform/slurm-gcp -C 5.12.0 \
+              -U https://github.com/GoogleCloudPlatform/slurm-gcp -C 6.5.13 \
```

Comment:
`6.6.0`

Reply:
The tag wasn't available at the time I performed manual testing. The integration test still has known gaps so I prefer to document this as a task to be prioritized soon.